components/TextFields/tests/snapshot/MDCTextFieldFullWidthControllerSnapshotTests.m

```diff
@@ -0,0 +1,215 @@
+// Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY MDCTextFieldSnapshotTestsIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "MDCSnapshotTestCase.h"
+#import "MDCTextFieldSnapshotTestsStrings.h"
+#import "MaterialTextFields.h"
+
+@interface MDCTextFullWidthControllerSnapshotTests : MDCSnapshotTestCase
+@property(nonatomic, strong) MDCTextField *textField;
+@property(nonatomic, strong) MDCTextInputControllerFullWidth *textFieldController;
+@end
+
+@implementation MDCTextFullWidthControllerSnapshotTests
+
+- (void)setUp {
+  [super setUp];
+
+  self.textField = [[MDCTextField alloc] init];
+  self.textFieldController =
+      [[MDCTextInputControllerFullWidth alloc] initWithTextInput:self.textField];
+}
+
+- (void)tearDown {
+  self.textFieldController = nil;
+  self.textField = nil;
+
+  [super tearDown];
+}
+
+#pragma mark - Helpers
+
+- (void)triggerTextFieldLayout {
+  CGSize aSize = [self.textField sizeThatFits:CGSizeMake(300, INFINITY)];
+  self.textField.bounds = CGRectMake(0, 0, aSize.width, aSize.height);
+  [self.textField layoutIfNeeded];
+}
+
+- (void)generateSnapshotAndVerify {
+  [self triggerTextFieldLayout];
+  UIView *snapshotView = [self addBackgroundViewToView:self.textField];
+
+  // Perform the actual verification.
+  [self snapshotVerifyView:snapshotView];
+}
+
+#pragma mark - Tests
+
+- (void)testOutlinedTextFieldEmpty {
```

Comment:
Names need to be updated? It looks like these aren’t outlined text fields.

Reply:
Yeah I believe a find/replace is needed

Reply:
( -_-) Oops.... Done